backend/src/main/java/com/google/rolecall/services/PerformanceServices.java

```diff
@@ -202,7 +202,9 @@ private PerformanceSection addPerformanceCastMember(PerformanceSection performan
     User user = userService.getUser(memberInfo.userId());
 
     PerformanceCastMember member = PerformanceCastMember.newBuilder()
-        .setOrder(memberInfo.order())
+        // .setOrder(memberInfo.order())
+        // YHE
```

Comment:
Wondering the need for leaving comment with initials? I think all of these comments can be removed since we can see who changed what/when using standard github features.

Reply:
Done!